bin/rules.d/a8-m3.rules

```diff
@@ -1,4 +1,4 @@
 # Open A8 M3 node
-SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ENV{ID_SERIAL}=="IoT-LAB_A8-M3",         ENV{ID_USB_INTERFACE_NUM}=="01",  SYMLINK+="ttyA8_M3 iotlab/ttyA8_M3"
+SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ENV{ID_SERIAL}=="IoT-LAB_A8-M3",         ENV{ID_USB_INTERFACE_NUM}=="01",  SYMLINK+="ttyA8_M3"
```

Comment:
I think there is no need for changing this. I would even think it will useful in the future with custom open node behind the Linux node. Because the TTY will always be `/dev/iotlab/tty<something>`